### DISCUSSION TITLE
Shiny app to display ArchR results

### DISCUSSION THREAD

**Discussion by sylestiel** [GreenleafLab/ArchR#1144] [Feature Requests]
Hi,

 Any suggestions on how to generate a Shiny app to display the results of our ArchR analyses and make it interactive for publication purposes?

Thanks!

**Comment by rcorces** [comment]
this is on the roadmap eventually to have as an integrated component of ArchR. In the meantime someone else made this - https://pubmed.ncbi.nlm.nih.gov/34586377/
I cant say how well it works and i dont know what assumptions it makes and how changes to ArchR in the future might break things on their end.

**Reply by sylestiel** [reply]
Thanks!
